src/org/traccar/protocol/Gl200ProtocolDecoder.java

```diff
@@ -660,13 +717,16 @@ protected Object decode(
                 case "IDA":
                     result = decodeIda(channel, remoteAddress, sentence);
                     break;
+                case "VER":
+                    result = decodeVer(channel, remoteAddress, sentence);
+                    break;
                 default:
                     result = decodeOther(channel, remoteAddress, sentence, type);
                     break;
             }
 
             if (result == null) {
-                result = decodeBasic(channel, remoteAddress, sentence);
+                result = decodeBasic(channel, remoteAddress, sentence, type);
```

Comment:
Ideally we would want to get rid of this method eventually. That's basically a backup for all un-handled types. It's just a comment, no action needed at this point.